README.md

```diff
@@ -38,13 +42,14 @@ GPM/IMERGHH and MERRA collections.
   service code and Docker image to GitHub.
 * bin - A directory containing utility scripts to build the service and test
   images.
+* dev_requirements.txt - A list of Python package dependencies for library development.
```

Comment:
Super nit: The file name has a dash not an underscore.

Reply:
61fc66a